src/gallery.js

```diff
@@ -59,50 +135,39 @@ const hideFullscreenGallery = ( galleryContainer ) => {
 			}
 		} )
 
-		fullscreenGallery.classList.add( 'wp-gallery-popup' )
-		top.classList.add( 'wp-gallery-popup-top' )
-		main.classList.add( 'wp-gallery-popup-main' )
-		bottom.classList.add( 'wp-gallery-popup-bottom' )
+		galleryContainer.insertAdjacentHTML( 'beforeend', fullscreenGallery )
+
+		let image = galleryContainer.querySelector( 'img' ),
+			caption = galleryContainer.querySelector( '.wp-gallery-popup-caption' ),
+			loading = galleryContainer.querySelector( '.wp-gallery-popup-loading' ),
+			nextButton = galleryContainer.querySelectorAll( '.wp-gallery-popup-button.next' )[ 0 ],
```

Comment:
Is there a difference between `querySelectorAll(...)[0]` and `querySelector(...)` ?

Reply:
Hmm not if we query only `next`/`previous`, that's more readable 👍